deps/wazuh_testing/wazuh_testing/tools/performance/statistic.py

```diff
@@ -161,7 +161,7 @@ def _parse_api_data(self):
             try:
                 token_response = requests.get(API_URL + TOKEN_ENDPOINT, verify=False,
                                               auth=requests.auth.HTTPBasicAuth("wazuh", "wazuh"))
-                if token_response.status_code != 200:
+                if token_response.status_code != '200':
```

Comment:
Status code it's an integer, not a string

Reply:
fixed in https://github.com/wazuh/wazuh-qa/pull/5014/commits/bb491b64f3388481d0263f9c3c8ae4289a2b022a